data/landing.yaml

```diff
@@ -83,26 +56,26 @@ components:
+    - title: MCP Gateway
+      icon: build
+      description: MCP Gateway is an Envoy-based gateway for connecting agents to tools via the Model Context Protocol (MCP). It provides centralized routing for multiple MCP servers and "virtual MCP servers" that group tools from different sources, enabling precise control over agent tool access and policy enforcement. A Kubernetes-native control plane adds declarative management via custom CRDs, automatic backend discovery through HTTPRoute integration, and Kuadrant support for authorization and token exchange policies.
       ctaLink:
         text: Learn more
-        url: https://github.com/kagenti/kagenti-operator/tree/main/kagenti-operator
+        url: https://github.com/kagenti/mcp-gateway
 
-    - title: MCP Gateway
+    - title: Agent Identity and Authorization Guard
       icon: key
-      description: MCP Gateway is a Kubernetes-native, Envoy-based gateway designed to streamline access to multiple Model Context Protocol (MCP) servers. It leverages Kubernetes APIs like the MCPServer CRD for declarative server management, HTTPRoute integration for automatic backend discovery, and an operator that keeps your configuration in sync. With flexible policy support, including integration with Kuadrant or any Envoy-compatible policy engine, MCP Gateway empowers you to build scalable, secure, and policy-driven AI infrastructure with ease.
+      description: Agent Identity and Authorization Guard includes sidecars that provide identity-based, delegated access control for agents to tools, transparent to agent code. They implement agent identity using SPIFFE/SPIRE, dynamic OAuth client registration, token exchange with fine-grained scoping, and secret retrieval from vault for external tool integration. Together they ensure zero-trust security for agents and allow users to precisely control what agents can do on their behalf.
       ctaLink:
         text: Learn more
-        url: https://github.com/kagenti/mcp-gateway
+        url: https://github.com/kagenti/kagenti?tab=readme-ov-file#agent-and-tool-authorization-pattern
 
-    - title: Identity and Tool Authorization Pattern
-      icon: build
-      description: Identity and Tool Authorization Pattern brings zero-trust security to the Agentic Platform by enforcing continuous authentication and fine-grained authorization for both human and machine identities. Built on SPIFFE/SPIRE and integrated with Keycloak, it replaces static credentials with dynamic, short-lived tokens, ensuring secure identity propagation and least-privilege access across services. Through structured token exchanges and real-time policy enforcement, this pattern enables secure delegation, robust machine identity management, and continuous verification—laying the foundation for scalable, production-ready agentic workflows.
+    - title: Kagenti Life-cycle Operators
```

Comment:
I know both life-cycle and lifecycle can be used, but I understand life-cycle is more for scientific publications while lifecycle is more informal (e.g., marketing language) - so I suggest to use Lifecycle